.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you remove this file from the PR? Thanks

Reply:
Sorry for that. It is for disabling styles from plugin to not destroy how is done